lib/Sema/TypeCheckDecl.cpp

```diff
@@ -2176,6 +2176,7 @@ InterfaceTypeRequest::evaluate(Evaluator &eval, ValueDecl *D) const {
   case DeclKind::Module:
   case DeclKind::OpaqueType:
   case DeclKind::GenericTypeParam:
+  case DeclKind::Error:
```

Comment:
It may eventually become desirable to compute the interface type of an `ErrorDecl`. Considering they have no name, I do not know how such a situation would arise today. But the obvious implementation here would be to return `ErrorType`.

Reply:
I mean, hypothetically there's nothing stopping `ErrorDecl` from showing up in the member list of any `DeclContext`. There's all kinds of ways one could imagine an analysis that (transitively) iterates over those members to eventually ask for the interface type at some point...